host/src/main/java/org/openjdk/skara/host/Host.java

```diff
@@ -31,6 +31,7 @@
     HostUserDetails getUserDetails(String username);
     HostUserDetails getCurrentUserDetails();
     boolean supportsReviewBody();
+    boolean isMemberOf(long groupId, HostUserDetails user);
```

Comment:
You may want to make the groupId parameter a String instead, there are hosts out there that we may want to support that don't use numerical ids.

Reply:
Thanks, updated!